libevm/VMConfig.h

```diff
@@ -22,17 +22,28 @@ namespace eth
 
 ///////////////////////////////////////////////////////////////////////////////
 //
-// interpreter configuration macros for optimizations and tracing
+// interpreter configuration macros for development, optimizations and tracing
+//
+// EIP_615                - subroutines and static jumps
+// EIP_616                - SIMD
 //
 // EVM_SWITCH_DISPATCH    - dispatch via loop and switch
 // EVM_JUMP_DISPATCH      - dispatch via a jump table - available only on GCC
 //
-// EVM_USE_CONSTANT_POOL  - 256 constants unpacked and ready to assign to stack
+// EVM_USE_CONSTANT_POOL  - constants unpacked and ready to assign to stack
 //
-// EVM_REPLACE_CONST_JUMP - with pre-verified jumps to save runtime lookup
+// EVM_REPLACE_CONST_JUMP - pre-verified jumps to save runtime lookup
 //
 // EVM_TRACE              - provides various levels of tracing
 
+#ifndef EIP_615
+	#define EIP_615 false
```

Comment:
Looking at the code again, I still like this pattern.  It allows for external configuration (via cmake or whatever) - if a macro has a definition when compiled it is respected, and if it doesn't an appropriate default is chosen.  During development I just change the defaults to quickly switch configurations.